biocrnpyler/component.py

```diff
@@ -141,6 +121,13 @@ def add_attribute(self, attribute: str):
 
 
     def update_parameters(self, parameter_file = None, parameters = None, parameter_database = None, overwrite_parameters = True):
+        """Updates the ParameterDatabase inside a Component
+
+        Possible inputs:
+            parameter_file (string)
+            parameters (dict)
+            parameter_database (ParameterDatabase)        
+        """
 
         if parameter_file is not None:
             self.parameter_database.load_parameters_from_file.load_parameters_from_dictionary(parameter_file, overwrite_parameters = overwrite_parameters)
```

Comment:
Not part of this PR, but this line looks fishy.

Reply:
good spot - fixed it